packages/esm-patient-chart-app/src/ui-components/patient-chart.component.scss

```diff
@@ -58,3 +60,7 @@ $actionPanelExpandedOffset: $actionNavOffset + $actionPanelOffset;
   padding-right: $actionNavOffset;
   flex-direction: column;
 }
+
+.marginRight {
+  margin-right: 35%;
```

Comment:
This does not feel right. It should be implicitly right.

Usually, this is where flex box shines.

Reply:
Tried using flex but since we are using `position: fixed` it is taken out of the normal flow and is therefore not applicable

Reply:
Yes - we should remove these fixed positions. Just saying.

(My tendency towards flexbox was not "keep everything as is", but rather "lets change it to apply a better layouting mechanism"...)